_posts/2019-06-23-qsharp-triple-slash.md

```diff
@@ -0,0 +1,62 @@
+# Autogenerating Documentation Comments for Q# in Visual Studio

```

Comment:
The blog posts should have some header yaml to help the static site generator format it correctly.

Also if you want it to show on the sidebar your author profile, you can start the `authors.yml` file as specified in [the theme guide](https://mmistakes.github.io/minimal-mistakes/docs/authors/) and then add to this header the line `author: jclapis`

```suggestion
---
title: "Autogenerating Documentation Comments for Q# in Visual Studio"
date: 2019-06-24T09:22-06:00
categories:
  - blog
tags:
  - contributed-post
  - visual-studio
  - documentation
---
```